docs/content/guides/getting-started/installation.md

```diff
@@ -222,11 +242,43 @@ const hot = new Handsontable(container, {
 :::
 :::
 
+:::
+
 ::: only-for react
+<br/>
```

Comment:
Without this break header below wont apply as a header. I tried few different configurations without success.

Reply:
I think I found a solution. I pushed it to this PR: ec31a6c454bff2a533ff89499f4f0b4d01798b20. Please revert if you disagree.